spec/tasks/load_data_test_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+require 'rake'
+Rails.application.load_tasks
+
+RSpec.describe "load_csv task" do
+  
+  before :each do 
+    Rake::Task[:load_csv].invoke("_test")
+    # Rake::Task[:load_csv].invoke
+  end
+
+  xit "Models Created" do
```

Comment:
It works, currently skipping because it will fail if there are other tests. need to look into this